src/EFCore.PG.NTS/Query/ExpressionTranslators/Internal/NpgsqlNetTopologySuiteMethodCallTranslatorPlugin.cs

```diff
@@ -116,4 +119,37 @@ SqlExpression OneBased(SqlExpression arg)
                     : (SqlExpression)_sqlExpressionFactory.Add(arg, _sqlExpressionFactory.Constant(1));
         }
     }
+
+    /// <summary>
+    /// Translates functions operating on types implementing the <see cref="IGeometry"/> interface.
+    /// </summary>
+    public class NpgsqlGeometryFunctionTranslator : IMethodCallTranslator
+    {
+        static readonly Dictionary<MethodInfo, string> Functions = new Dictionary<MethodInfo, string>
+        {
+            [GetRuntimeMethod(nameof(NpgsqlNetTopologySuiteDbFunctionsExtensions.Transform), new[] { typeof(DbFunctions), typeof(Geometry), typeof(int) })] = "ST_Transform",
+        };
+
+        static MethodInfo GetRuntimeMethod(string name, params Type[] parameters)
```

Comment:
I like proper factoring, but this seems a bit excessive :) Let's just call the GetRuntimeMethod directly above...